src/components/settings-components/SettingsView.vue

```diff
@@ -314,6 +314,9 @@ import CdnProvider from '../../providers/generic/connection/CdnProvider';
                 'Refresh online mod list',
                 'Check for any new mod releases.',
                 async () => {
+                        if (this.$store.getters['download/activeDownloadCount'] > 0) {
```

Comment:
Here too it might make sense to prefer to show the current status of update if mod downloads and mod list refresh are going on at the same time. Note that while the refresh can't be started while mods are downloading, user can still start downloads while the refresh is in progress, so both can be happening in the same time.

Reply:
Oops, should be fixed now 👌